inference-engine/src/transformations/src/transformations/op_conversions/convert_ctc_greedy_decoder_v6_to_v1.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "itt.hpp"
+
+#include "transformations/op_conversions/convert_ctc_greedy_decoder_v6_to_v1.hpp"
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertCTCGreedyDecoderV6ToV1, "ConvertCTCGreedyDecoderV6ToV1", 0);
+
+ngraph::pass::ConvertCTCGreedyDecoderV6ToV1::ConvertCTCGreedyDecoderV6ToV1() {
```

Comment:
Looks like this transformation does nothing with v1 versoin as it alway use v6. So I suggest to rename transformation to `SimplifyCTCGreedyDecoder`